src/transforms/eigenanalysis.jl

```diff
@@ -3,9 +3,10 @@
 # ------------------------------------------------------------------
 
 """
-    EigenAnalysis(proj)
+    EigenAnalysis(proj; ndim=nothing)
```

Comment:
Let's make it a positional argument. This EigenAnalysis transform is more low-level and someone using it can simply pass a second argument if necessary without typing ndim.